pkg/reconciler/shared/tektonconfig/controller.go

```diff
@@ -95,3 +101,14 @@ func NewExtensibleController(generator common.ExtensionGenerator) injection.Cont
 		return impl
 	}
 }
+
+// enqueueCustomName adds an event with name `config` in work queue so that
+// whenever a namespace event occurs, the TektonConfig reconciler get triggered.
+// This is required because we want to get our TektonConfig reconciler triggered
+// for already existing and new namespaces, without manual intervention like adding
+// a label/annotation on namespace to make it manageable by Tekton controller.
+func enqueueCustomName(impl *controller.Impl, name string) func(obj interface{}) {
+	return func(obj interface{}) {
```

Comment:
previously we were filtering out namespace kube/openshift-
```
if err == nil && !nsRegex.MatchString(object.GetName()) {
			impl.EnqueueKey(types.NamespacedName{Namespace: "", Name: name})
		}
```
do we don't need to now?

Reply:
previously, it was as below. we use to filter out namespaces `kube|openshift-` and wouldn't trigger on events on those namespaces
but now in the pr it the filter is removed so just wanna know do we need to watch event on all namespaces? 
```
func enqueueCustomName(impl *controller.Impl, name string) func(obj interface{}) {
	return func(obj interface{}) {
		object, err := kmeta.DeletionHandlingAccessor(obj)
		if err == nil && !nsRegex.MatchString(object.GetName()) {
			impl.EnqueueKey(types.NamespacedName{Namespace: "", Name: name})
		}
	}
}
```

Reply:
@sm43 Thanks for this catch. We need it. In multiple updates this got changed :+1: